packages/discord.js/src/util/Util.js

```diff
@@ -489,15 +489,15 @@ class Util extends null {
       .replace(/<@!?[0-9]+>/g, input => {
         const id = input.replace(/<|!|>|@/g, '');
         if (channel.type === ChannelType.DM) {
-          const user = channel.client.users.cache.get(id);
+          const user = channel.recipient;
           return user ? `@${user.username}` : input;
         }
 
         const member = channel.guild.members.cache.get(id);
         if (member) {
           return `@${member.displayName}`;
         } else {
-          const user = channel.client.users.cache.get(id);
+          const user = channel.guild?.members?.cache.get(id)?.user;
```

Comment:
This is literally the same call as above, only with extra `?.` to something we already know doesn't exist